types/openlayers/index.d.ts

```diff
@@ -9906,6 +9907,29 @@ declare module ol {
     }
 
     /**
+     * Object literal with options for the {@link ol.Sphere.getLength} or
+     * {@link ol.Sphere.getArea} functions.
+     */
+    interface SphereMetricOptions {
+
+        /**
+         * Projection of the geometry.  By default, the geometry is assumed to be in
+         * EPSG:3857 (Web Mercator).
+         */
+        projection?: proj.Projection;
```

Comment:
According to the documentation, projection is a `ol.ProjectionLike`.

Reply:
you're right, I'll change that thanks